clang/lib/Sema/SemaDecl.cpp

```diff
@@ -3217,6 +3217,22 @@ static void adjustDeclContextForDeclaratorDecl(DeclaratorDecl *NewD,
     FixSemaDC(VD->getDescribedVarTemplate());
 }
 
+template <typename AttributeName>
```

Comment:
I think it is more type of attribute rather than name. i.e.
```suggestion
template <typename AttributeType>
```

Reply:
Agreed